src/app/api/getPoints.ts

```diff
@@ -0,0 +1,70 @@
+import { encode } from "url-safe-base64";
+
+import { Pagination } from "../types/api";
+
+import { pointsApiWrapper } from "./pointsApiWrapper";
+
+export interface StakerPoints {
+  staker_btc_pk: string;
+  points: number;
+}
+
+export interface DelegationPoints {
+  staking_tx_hash_hex: string;
+  staker: {
+    pk: string;
+    points: number;
+  };
+  finality_provider: {
+    pk: string;
+    points: number;
+  };
+  staking_height: number;
+  unbonding_height: number | null;
+  expiry_height: number;
+}
+
+export interface PaginatedDelegationsPoints {
+  data: DelegationPoints[];
+  pagination: Pagination;
+}
+
+export interface DelegationsPoints {
+  data: DelegationPoints[];
+}
+
+export const getStakersPoints = async (
```

Comment:
Wonder whether we should have an API for finality provider points. I guess the front-end doesn't need it, but fps might ask cc @jrwbabylonlab